config/ng-config.json

```diff
@@ -6,7 +6,7 @@
       "dateDisplayFormat": "DD-MMM-YYYY",
       "timeDisplayFormat": "hh:mm",
       "locationUrl": "/openmrs/ws/rest/v1/location",
-      "bahmniSearchUrl": "/openmrs/ws/rest/v1/bahmnicore/search",
```

Comment:
@sivareddytw we didn't want to use **bahmnicommons** ? instead of "bahmni" in the API endpoint ?

Reply:
yes...